RELEASE_NOTES

```diff
@@ -3,17 +3,7 @@
 ----------------------------------------------------------------------
 
 ### CH4
- * Embedded modules
-   - The release is thoroughly tested with embedded libfabric and ucx.
-     Using externally installed libfabric or ucx is likely to succeed
-     but less tested.
-
- * Build Issues
-   - Embedded ucx or libfabric builds will not succeed with Solaris
-     compilers.
-
  * Dynamic process support
-   - Requires fi_tagged capability in OFI netmod
```

Comment:
All above are still true, although I don't mind not mentioning them.

Reply:
OK, I thought we have removed the tagged requirement, but I will add it back.